astropy/stats/sigma_clipping.py

```diff
@@ -144,21 +150,14 @@ class SigmaClip:
 
     cenfunc : {'median', 'mean'} or callable, optional
         The statistic or callable function/object used to compute the
-        center value for the clipping.  If set to ``'median'`` or
-        ``'mean'`` then having the optional `bottleneck`_ package
-        installed will result in the best performance.  If using a
-        callable function/object and the ``axis`` keyword is used, then
-        it must be callable that can ignore NaNs (e.g., `numpy.nanmean`)
-        and has an ``axis`` keyword to return an array with axis
-        dimension(s) removed.  The default is ``'median'``.
-
-        .. _bottleneck:  https://github.com/pydata/bottleneck
+        center value for the clipping. If using a callable function/object and
+        the ``axis`` keyword is used, then it must be callable that can ignore
+        NaNs (e.g., `numpy.nanmean`) and has an ``axis`` keyword to return an
+        array with axis dimension(s) removed.  The default is ``'median'``.
 
-    stdfunc : {'std'} or callable, optional
+    stdfunc : {'std', 'mad_std'} or callable, optional
         The statistic or callable function/object used to compute the
-        standard deviation about the center value.  If set to ``'std'``
-        then having the optional `bottleneck`_ package installed will
-        result in the best performance.  If using a callable
+        standard deviation about the center value. If using a callable
         function/object and the ``axis`` keyword is used, then it must
         be callable that can ignore NaNs (e.g., `numpy.nanstd`) and has
```

Comment:
And same (so this existed before). If you wish you can also just insert "a" before "callable", but I think my suggestion above is better.

Reply:
Fixed